packages/server/src/insights/insight-classes.ts

```diff
@@ -10,6 +10,7 @@ import {
 import { UserCourseModel } from 'profile/user-course.entity';
 import { UserModel } from 'profile/user.entity';
 import { QuestionModel } from 'question/question.entity';
+import { strategy } from 'sharp';
```

Comment:
Is this needed?

Reply:
lol idk how that got in there, I'll remove it